kotlin/android/app/src/main/AndroidManifest.xml

```diff
@@ -43,19 +43,9 @@
 
         <activity
             android:name="dev.firezone.android.features.applink.presentation.AppLinkHandlerActivity"
-            android:exported="true"
+            android:exported="false"
```

Comment:
```suggestion
            android:exported="true"
```
The `exported` attribute exposes, the activity to be launched from outside the app's scope, for ex: launch the app or deeplink. As the AppLinkHandlerActivity handles the applink/deeplink, exported `true` is required.

Reply:
Thanks for the catch. Apologies for any misunderstanding, but does this mean that any app could open an AppLink and set the JWT in the Firezone app?

I was under the impression that the opened web view was in our app's scope, since it was launched within our app (this is how it works on Apple for example)

Reply:
Resolved over Slack